src/tests/integration/mapping.test.ts

```diff
@@ -42,4 +40,36 @@ describe('/api', () =>{
       expect(data.fullDomain).toEqual(`${subDomain}.${domain}`)
     })
   })
+
+  it('Delete mapping', async()=>{
+    const subDomain = `delete${uuidv4()}`
+    const domain = 'albertow'
+    const port = '4500'
+    await fetch(`${apiURL}/api/mappings`, {
+      method: 'POST', 
+      headers: {
+        authorization: ADMIN,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        domain,
+        subDomain,
+        port,
+      })
+    }).then(r => r.json()).then(async (data)=>{
+      await fetch(`${apiURL}/api/mappings/delete/${data.id}`, {
+        method: 'DELETE', 
+        headers: {
+          authorization: ADMIN,
+          'Content-Type': 'application/json'
+        }
+      }).then(r=>r.json()).then((del)=>{
+        expect(del.port).toEqual(port)
+        expect(del.subDomain).toEqual(subDomain)
+        expect(del.domain).toEqual(domain)
+        expect(del.fullDomain).toEqual(`${subDomain}.${domain}`)
+        expect(del.id).toEqual(data.id)
+      })
```

Comment:
could you do another fetch after this.... to ` GET /api/mappings/${data.id}` and make sure the response is null? This way we make sure its 100% deleted. Sometimes, just because `delete` sends back a correct response, doesn't mean it actually got deleted.